src/jit/codegenxarch.cpp

```diff
@@ -773,8 +773,8 @@ void CodeGen::genCodeForBinary(GenTree* treeNode)
     GenTreePtr op1 = treeNode->gtGetOp1();
     GenTreePtr op2 = treeNode->gtGetOp2();
 
-    // Commutative operations can mark op1 as contained to generate "op reg, memop/immed"
-    if (op1->isContained())
+    // Commutative operations can mark op1 as contained or used from spill to generate "op reg, memop/immed"
```

Comment:
Note that the code generator still cares about the notion of containment - that's what determine whether it generates code for any given node. And a contained immediate is still a valid concept.